maps-compose/src/main/java/com/google/maps/android/compose/CameraPositionState.kt

```diff
@@ -235,17 +241,23 @@ class CameraPositionState(
     private fun performAnimateCameraLocked(
         map: GoogleMap,
         update: CameraUpdate,
+        durationMs: Int?,
```

Comment:
This no longer needs to be nullable, right?

Reply:
Yep, the conditional should also check for `MAX_VALUE` 😓  Done in 3f85be9